timon/plugins/base.py

```diff
@@ -47,3 +49,22 @@ async def stop_plugin(self):
     async def stop(self):
         raise NotImplementedError(
             "Plugin %s doesn't have a start method implemented", self.name)
+
+
+class OnDbStorePlugin:
+    """ if a plugin also derives from this class,
+        it will be invoked when a prob's state was updated and is being stored
+
```

Comment:
`prob` ican also be used as a shortcut for `problem`, so probably better to write `probe`.

Reply:
yes, typo, I missed the 'e'

of course this file talks a lot about "problems" having a "state" that can be "updated"